src/compiler/_namespaces/ts.ts

```diff
@@ -72,3 +72,11 @@ import * as moduleSpecifiers from "./ts.moduleSpecifiers";
 export { moduleSpecifiers };
 import * as performance from "./ts.performance";
 export { performance };
+/** @internal */
+import * as NodeConstructors from "./ts.NodeConstructors";
+/** @internal */
+export { NodeConstructors };
+/** @internal */
+import * as ObjectConstructors from "./ts.ObjectConstructors";
+/** @internal */
+export { ObjectConstructors };

```

Comment:
Does this work? My understanding after writing the DTS bundler was that this didn't work, so the bundler instead just recognizes empty namespaces as being fully internal and removes them.

Reply:
Yes, it does:

![image](https://user-images.githubusercontent.com/3902892/208256059-63703a23-4113-47c9-84e9-7f2146dd6d02.png)

Though you have to annotate both the import and export if you're splitting it up:

![image](https://user-images.githubusercontent.com/3902892/208256070-e90a1dea-3d21-44a6-a95a-9fbbe19e332e.png)

Reply:
That's our d.ts emit, but since we post-process, it's more about whether or not the comment makes it out into the otuput, which it doesn't.

Reply:
In any case, I don't think we should add anything to these namespaces, instead opting for direct imports, even across projects.

Reply:
(I'll stop commenting though, I know that this is WIP and you're just trying to get it working.)